src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -33,7 +34,7 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
     private _portal: TemplatePortal;
 
     /** Store the reference to the overlay */
-    private _overlayRef: OverlayRef;
+    _overlayRef: OverlayRef;
```

Comment:
Could we keep this private and add a getter specifically for aria controls?

The main reason is `_overlayRef` really should be a marked as possibly undefined `private _overlayRef?: OverlayRef;`, this might require a few changes to other places in the code, but `this._overlayRef.overlayElement.id` could cause errors if we are trying to access `overlayElement` of undefined. Whereas if we have a dedicated getter for this we can use nullish coelescing:

```typescript
get ariaControls(): string | null {
    return this.menu?.isMenuOpen ? this._overlayRef?. overlayElement.id : null;
}
```